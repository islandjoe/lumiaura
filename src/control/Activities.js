import React from 'react'
import JobType from './activities/JobType'
import Activities from '../layout/Activities.css'

const activities =(props)=> {
  return (
    <div className={Activities}>
      <JobType type="road" label="Roads"/>
      <JobType type="cycling" label="Cycling"/>
      <JobType type="footpath" label="Footpath"/>
    </div>
  )
}

export default activities