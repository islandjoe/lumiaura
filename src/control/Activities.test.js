import React from 'react'
import renderer from 'react-test-renderer'
import {configure} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import Activities from './Activities'

configure({ adapter: new Adapter() })

describe('<Activities/>', ()=> {

  test('should render', ()=> {
    const c = renderer
      .create(<Activities/>)
      .toJSON()

    expect(c).toMatchSnapshot()
  })

})