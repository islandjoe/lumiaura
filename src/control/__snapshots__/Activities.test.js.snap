// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Activities/> should render 1`] = `
Array [
  <label
    className="JobType ui label"
  >
    <input
      className="ui checkbox"
      name="road"
      type="checkbox"
    />
    Road
  </label>,
  <label
    className="JobType ui label"
  >
    <input
      className="ui checkbox"
      name="cycling"
      type="checkbox"
    />
    Bike Lanes
  </label>,
  <label
    className="JobType ui label"
  >
    <input
      className="ui checkbox"
      name="footpath"
      type="checkbox"
    />
    Pedestrian/Footpath
  </label>,
]
`;
